libMBIN/Source/NMS/GameComponents/GcUAProtectedLocations.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+
+using libMBIN.NMS.Toolkit;
+using libMBIN.NMS.GameComponents;
+
+namespace libMBIN.NMS.GameComponents
+{
+    [NMS(Size = 0x18, GUID = 0x694C9DF94CECC153, NameHash = 0xD522308022E072F4)]
+    public class GcUAProtectedLocations : NMSTemplate
+    {
+        /* 0x00 */ public Vector3f Location;
```

Comment:
I sadly don't have the most recent exe decompiled to check, but I think that there is something wrong with this?? The other protected location class has size 0x20 because it contains a vector3f which is alignment 0x10, so the class will also have it's size % 0x10. This class however has a size 0x18 which would seem to indicate that there can't be anything with alignment greater than 0x8 in it... Again, I can't check in exe until I decompile the latest version...